echo.go

```diff
@@ -144,3 +149,9 @@ func showResult(c echo.Context) error {
 func (c listResultsCtx) LinkForID(id string) string {
 	return c.echo.Reverse("showResult", id)
 }
+
+var anchorRegex = regexp.MustCompile(`[^a-zA-Z0-9._:/'()-]`)
+
+func anchorForResult(pkg, name string) string {
+	return string(anchorRegex.ReplaceAll([]byte(pkg+"/"+name), []byte{'_'}))
+}
```

Comment:
Are we removing special chars as to not break url encoding?

Reply:
If so, can't we `url.QueryEscape` the string?

Reply:
yeah, we need some sort of filtering here since the string is used verbatim as the value for the attribute. there's probably a better way to do this but figuring out how to do this at all took the vast majority of the time I spent on this (html/template sure implies that there are pre-defined things that do this but they actually aren't).

Reply:
since our comments crossed, this is _not_ just used in the context of a query string. this is also used as the ID on the table row itself, so it does not get URL escaped there but we definitely can't have `"`, at least.